config/initializers/devise.rb

```diff
@@ -236,9 +236,8 @@
   config.sign_out_via = :delete
 
   # ==> OmniAuth
-  # Add a new OmniAuth provider. Check the wiki for more information on setting
-  # up on your models and hooks.
-  # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
+  # Add a new OmniAuth provider.
+  config.omniauth :urs, ENV['CLIENT_ID'], ENV['CLIENT_SECRET']
```

Comment:
I'm not sure if this is the right place, but I think with both this PR and the gem, we need to address the issue we discussed of connecting to the appropriate URS environment (SIT, UAT, PROD), as this code moves through our environments.

Reply:
Another ticket has been created for this.